src/documentation/examples/Modal/focus_reset.jsx

```diff
@@ -18,7 +18,7 @@ export default function ModalTest() {
       />
       { modalOpen && (
         <Modal
-          action={{ label: "Don’t Panic!" }}
+          action={{ label: "Don’t Panic!", callback: () => {} }}
```

Comment:
Hey! @gmzjuliana (cc @dace)
I'm adding this callback here to the example to view the documentation without the error. Although this doesn't "fix" it, I think it might be an intentional behavior, but if it's not, I think it makes more sense to fix it in a different PR since it deviates from the real objective of this PR.

Reply:
Thanks @MayaUribe 🙏 👍  Apologies for breaking the documentation. Yeah, you are correct that the behavior is intentional. I can't think of a case where we would need a button without giving it some functionality via a callback so that is why I added a check that requires both a label **AND** a callback.